cmd/kots/cli/install.go

```diff
@@ -167,6 +167,7 @@ func InstallCmd() *cobra.Command {
 				Context:                   v.GetString("context"),
 				SharedPassword:            sharedPassword,
 				ApplicationMetadata:       applicationMetadata,
+				AppSlug:                   appSlugFromArgs(args[0]),
```

Comment:
How about just saving it at as `UpstreamURI` since this could also be a helm chart instead of kots app